src/Callback.php

```diff
@@ -164,4 +176,77 @@ private function schedule_callback( $session_id ) {
 
 		return 0 !== $did_schedule;
 	}
+
+	/**
+	 * Handles notification callbacks
+	 *
+	 * @param $request
```

Comment:
Add the proper type for the definition. Should be for example `* @param \WP_REST_Request $request The REST request.`